### PR TITLE
[anomalies] added algo info for agile and basic

### PR DESCRIPTION
**Pull request by austian**
@githomin @StephenKappel

### CODE REVIEW

File: content/guides/anomalies.md [hunk 1]
```diff
@@ -56,10 +56,13 @@ Continue with steps (3) and (4) as you would for any other monitor.
 
 ### 3. Anomaly Detection Algorithms
 
-We currently offer two different anomaly detection algorithms. Both algorithms will follow the seasonal pattern of the series, and
-will not have their predictions be affected by short anomalies.
+We currently offer four different anomaly detection algorithms. All of the algorithms use past history when forming forecasts, however Robust, Agile, and Adaptive detect and use any additional seasonal structure present in the series when forming their predictions.
```
Comment (StephenKappel) [inline content/guides/anomalies.md]: I thought we were discontinuing adaptive?

I would axe the sentence starting with "All of ...". It's obvious that the algorithms use the past, and the differences are already described below.

File: content/guides/anomalies.md [hunk 2]
```diff
@@ -56,10 +56,13 @@ Continue with steps (3) and (4) as you would for any other monitor.
...
 
-* Robust: This algorithm is very stable and its predictions will remain constant even through longer-term anomalies. On the other hand, it will take longer to respond to intended level shifts (e.g., if the level of a metric shifts due to a code change.) This algorithm uses more data and can take longer to load the first time it is run.
+* Basic: This uses a simple lagging rolling quantile computation to make forecasts.
+
+* Agile: A robust version of the [SARIMA](https://en.wikipedia.org/wiki/Autoregressive_integrated_moving_average) algorithm. It will incorporate the immediate past into its predictions, allowing it to update quicker to level shifts at the expense of being less robust to longer-term anomalies.
```
Comment (StephenKappel) [inline content/guides/anomalies.md]: quicker -> quickly

longer-term -> recent, long-lasting

File: content/guides/anomalies.md [hunk 3]
```diff
@@ -56,10 +56,13 @@ Continue with steps (3) and (4) as you would for any other monitor.
...
+
+* Robust: A [seasonal-trend decomposition](https://en.wikipedia.org/wiki/Decomposition_of_time_series) algorithm. It is very stable and its predictions will remain constant even through longer-term anomalies. On the other hand, it will take longer to respond to intended level shifts (e.g., if the level of a metric shifts due to a code change.) This algorithm uses more data and can take longer to load the first time it is run.
```
Comment (StephenKappel) [inline content/guides/anomalies.md]: longer-term -> long-lasting

I would drop the part about taking longer to load. For most customers and most metrics, there is no noticeable difference.

**Comment by StephenKappel** [comment]
I'm wondering if we should write this in the present tense rather than the future tense. 

Instead of

> It is very stable and its predictions will remain constant even through longer-term anomalies. On the other hand, it will take longer to respond to intended level shifts....

this instead

> It is very stable and its predictions remain constant even though longer-term anomalies. On the other hand, it takes longer to respond to intended level shifts...

**Comment by austian** [comment]
I switched the tense.  I noticed it and changed it in the first paragraph, but not everywhere else.

File: content/guides/anomalies.md [hunk 4]
```diff
@@ -56,10 +56,13 @@ Continue with steps (3) and (4) as you would for any other monitor.
...
-will not have their predictions be affected by short anomalies.
+We currently offer four different anomaly detection algorithms.
 
-* Robust: This algorithm is very stable and its predictions will remain constant even through longer-term anomalies. On the other hand, it will take longer to respond to intended level shifts (e.g., if the level of a metric shifts due to a code change.) This algorithm uses more data and can take longer to load the first time it is run.
+* Basic: This uses a simple lagging rolling quantile computation to make forecasts.
+
+* Agile: A robust version of the [SARIMA](https://en.wikipedia.org/wiki/Autoregressive_integrated_moving_average) algorithm. It incorporates the immediate past into its predictions, allowing it to update quickly to level shifts at the expense of being less robust to recent, long-lasting anomalies.
+
+* Robust: A [seasonal-trend decomposition](https://en.wikipedia.org/wiki/Decomposition_of_time_series) algorithm. It is very stable and its predictions remain constant even through long-lasting anomalies. On the other hand, it takes longer to respond to intended level shifts (e.g., if the level of a metric shifts due to a code change.)
```
Comment (githomin) [inline content/guides/anomalies.md]: "anomalies, at the expense of taking longer"

**Pull request merged**